src/plugins/discourse/weights.js

```diff
@@ -1,9 +1,75 @@
 // @flow
 
 import {type NodeWeight} from "../../core/weights";
+import * as C from "../../util/combo";
+import * as NullUtil from "../../util/null";
 import {DEFAULT_TRUST_LEVEL_TO_WEIGHT} from "./createGraph";
 import {type User} from "./fetch";
 
+export type SerializedConfig = {|
+  +defaultWeight?: NodeWeight,
+  +weights?: {
+    [string]: NodeWeight,
+  },
+|};
+
+export type Config = {|
+  +defaultWeight: NodeWeight,
+  +weights: {
+    [string]: NodeWeight,
+  },
+|};
+
+export type CategoryConfig = Config;
+export type TagConfig = Config;
+
+export const DEFAULT_CONFIG: Config = {
```

Comment:
Why are `DEFAULT_CONFIG`, `upgrade`, and `parseCategoryId` exported?
They all seem like implementation details, and `DEFAULT_CONFIG` could be
mutated by poorly behaved clients.

Reply:
I was expecting to use them in tests in later PRs.